### DISCUSSION TITLE
OHBM 2022: Summary

### DISCUSSION THREAD

**Discussion by frheault** [frheault/tractography_file_format#62] [General]
Hello everyone,

This is a quick summary of TRX at OHBM:

First, there was good attendance at the Oral, Demo, and Poster sessions! A lot of people were interested in the fact that we convinced people to unite force BEFORE proposing a file format, a lot of questions came about the process rather than the file format itself.
They were curious about that because it seems like a unifying format often fails when they are proposed before gathering a group of collaborators, so I got a lot of positive feedback about the initiative being done 'the right way'. As expected, people were satisfied with the benchmarks, being slightly better was enough in most interactions. Features and uniformity in the field were most important.

I had a few good (in-person) discussions with @francopestilli , @arokem, @ayendiki and @lestropie. An evident pattern that came from these (and interaction at the poster) is that we absolutely need a working c++ implementation ASAP. In the fall I could push to use that implementation in MI-Brain, Freesurfer (@ayendiki) would probably be open to including a simple load/save supporting TRX. Hopefully, we can also use the same load/save for DSI-Studio (@frankyeh).

It was also exposed that mrtrix 'fancier' use of lazy-saving or lazy-loading would be the most complex integration of TRX, but I think a fair compromise would be to aim for at least a converter to make sure it works and dependencies work.

@ValHayot Do you have a guesstimate for a timeline of your c++ PR?

**Comment by ValHayot** [comment]
Hey, sorry, finally settled in from relocation. I'll wrap things up before the end of this month. Should have a full PR in soon!
